src/components/FilterPane/FilterPane.jsx

```diff
@@ -187,189 +193,213 @@ const FilterPane = ({ mermaidUserData }) => {
+                  className="clear-button"
+                >
+                  <IconClose />
+                </IconButton>
+              )}
+            </StyledDateInput>
 
-          <StyledDateInput>
-            <input
-              type="date"
-              value={formattedDate(sampleDateBefore)}
-              onChange={(e) => handleChangeSampleDateBefore(e.target.value)}
-            />
-            {sampleDateBefore && (
-              <IconButton
-                aria-label="clear date"
-                onClick={handleClearSampleDateBefore}
-                className="clear-button"
-              >
-                <IconClose />
-              </IconButton>
-            )}
-          </StyledDateInput>
+            <StyledDateInput>
+              <input
+                type="date"
+                value={formattedDate(sampleDateBefore)}
+                onChange={(e) => handleChangeSampleDateBefore(e.target.value)}
+              />
+              {sampleDateBefore && (
+                <IconButton
+                  aria-label="clear date"
+                  onClick={handleClearSampleDateBefore}
+                  className="clear-button"
+                >
+                  <IconClose />
+                </IconButton>
+              )}
+            </StyledDateInput>
+          </StyledDateInputContainer>
           {isAuthenticated ? (
             <>
               <StyledHeader>Your Data</StyledHeader>
-              <div>
-                <input
-                  type="checkbox"
-                  name="yourData"
-                  id="yourData"
-                  onChange={handleYourDataFilter}
-                  checked={showYourData}
-                />
-                <label htmlFor="yourData">{filterPane.yourData}</label>
-              </div>
-            </>
-          ) : null}
-          <StyledHeader>Data sharing</StyledHeader>
-          <div>
-            <input
-              type="checkbox"
-              name="dataSharing"
-              id="dataSharing"
-              onChange={handleDataSharingFilter}
-              checked={dataSharingFilter}
-            />
-            <label htmlFor="dataSharing">{filterPane.dataSharing}</label>
-          </div>
-          <StyledHeader>Methods</StyledHeader>
-          <StyledUnorderedList>
-            {COLLECTION_METHODS.map((method) => (
-              <li key={method.name}>
+              <StyledCategoryContainer>
                 <StyledClickableArea
-                  onClick={() =>
-                    handleMethodFilter({
-                      target: { name: method.name, checked: !methodFilters.includes(method.name) },
-                    })
-                  }
+                  onClick={() => handleYourDataFilter({ target: { checked: !showYourData } })}
                 >
                   <input
-                    id={method.name}
                     type="checkbox"
-                    name={method.name}
```

Comment:
Is there a reason from changing from using properties of `method` to hard coded values e.g. `yourData`